### DISCUSSION TITLE
Mocha before and after hooks not working in cypress cucumber project

### DISCUSSION THREAD

**Discussion by abhirammaleparampil** [cypress-io/cypress#18197] [Questions and Help]
Mocha before and after hooks not working in cypress cucumber project. beforeEach and afterEach is working as well.

**Comment by san-slysz** [comment]
https://github.com/TheBrainFamily/cypress-cucumber-preprocessor#before-and-after-hooks
Did you use Mocha before or cucumber before?
Also, share some code, else we can't help you.
